mne/viz/ica.py

```diff
@@ -206,11 +211,13 @@ def _plot_ica_sources_evoked(evoked, exclude, title, show):
     times = evoked.times * 1e3
 
     # plot unclassified sources
-    plt.plot(times, evoked.data.T, 'k')
+    if picks is None:
+        picks = np.arange(evoked.data.shape[0])
+    plt.plot(times, evoked.data[picks].T, 'k')
     for ii in exclude:
-        # use indexing to expose event related sources
-        plt.plot(times, evoked.data[ii].T, color='r', label='ICA %02d' % ii)
-
+        if ii in picks:
+            # use indexing to expose event related sources
+            plt.plot(times, evoked.data[ii].T, color='r', label='ICA %02d' % ii)
```

Comment:
here, the picks are the ICs you're interested in plotting. the chs are needed for the transformation in the step before this.